region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -307,6 +311,7 @@
         ]}
       ],
       "clearsObstacles": ["B"],
+      "flashSuitChecked": true,
```

Comment:
It's going to be so much harder to not lose it.

Reply:
Currently I'm counting all G-Mode strats as not allowing a flash suit. You will lose the suit when entering G-Mode, so it should be fine as long as we dont add ways to gain flash suits while already in G-Mode. 

I think any combinations of flash/blue/gmode are probably a very low priority for now (except generating suits with R-Mode).

Reply:
Someday we can change this if we model gaining a flash suit in G-Mode, but currently `canEnterGMode` requires `canEnterRMode` which requires `{"noFlashSuit": {}}`.